File: week1/mario-more/mario.c
// Recreates the pyramids from Mario for a given height

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    // Get user input(must be an int between 1-8)
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height < 1);

    // For each row, r
    for (int r = 0; r < height; r++)
    {
        // Print spaces
        for (int i = 0 ; i < height - r - 1; i++)
        {
            printf(" ");
        }

        // Print left hashes
        for (int i = 0 ; i < r + 1; i++)
        {
            printf("#");
        }

        // Print gap
        printf("  ");

        // Print right hashes
        for (int i = 0 ; i < r + 1; i++)
        {
            printf("#");
        }

        // Next row
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# Recreates pyramids in mario with #s

while True:
    try:
        height = int(input("Height: "))
        if height > 8 or height < 1:
             print("Please input an integer between 1 and 8")
        else:
            break
    except ValueError:
        print("Please input an integer between 1 and 8")

for i in range(height):
    left_space = height - i + 1
    print(" " * left_space + "#" * i + "  " + "#" * i)
